cmd/kubelet/app/server.go

```diff
@@ -413,8 +413,8 @@ func InitializeTLS(s *options.KubeletServer) (*server.TLSOptions, error) {
 	}
 	tlsOptions := &server.TLSOptions{
 		Config: &tls.Config{
-			// Change default from SSLv3 to TLSv1.0 (because of POODLE vulnerability).
-			MinVersion: tls.VersionTLS10,
+			// Change default from SSLv3 to TLSv1.2 (because of POODLE vulnerability).
```

Comment:
Please expand comment to list what is wrong with each version. i.e.

```
// Can't use SSLv3 because of POODLE
// Can't use TLSv1.0 because of xxxx
// Can't use TLSv1.1 because of xxxx
```